README.md

```diff
@@ -1,2 +1,14 @@
-# solidproject.org
-Website for solidproject.org
+# Solidproject.org
+
+Official website of the Solid Project
+
+## Local Environment Setup
+
+This website is built using [Jekyll](https://jekyllrb.com/), a static
+site generator.
+
+1. [Install](https://jekyllrb.com/docs/installation/) Ruby and Jekyll
```

Comment:
Possibly easier: install docker-compose, then run `docker-compose up`.

Reply:
@Vinnl can you add that as a second option?

Reply:
How about this?

```suggestion
If you have [Docker Compose](https://docs.docker.com/compose/) installed, view `docker-compose up` to start a local server to view the website.

Alternatively, you can install and run the required tooling manually as follows:
1. [Install](https://jekyllrb.com/docs/installation/) Ruby and Jekyll
```